README.md

```diff
@@ -4,14 +4,16 @@ GitHub actions to auto label a pull request based on committed files.
 
 ![intro](assets/intro.png)
```

Comment:
remove this one, otherwise the image is duplicated

Reply:
Done